salt/containerd/installed.sls

```diff
@@ -0,0 +1,24 @@
+{%- from "containerd/map.jinja" import containerd with context %}
+{%- from "repo/map.jinja" import repo with context %}
+
+include:
+  - repo
+  - runc
+
+Install containerd:
+  pkg.installed:
+    - name: containerd
+    - version: {{ containerd.version }}
+    - fromrepo: {{ repo.containerd.name }}
+    - require:
+      - pkgrepo: Configure Kubernetes repository
+      - pkg: Install runc
+
+Update container-selinux:
```

Comment:
Would it make sense to install this before `containerd`, so the upgrade isn't required and the `containerd` dependency on `container-selinux` is already met?

Reply:
Oh yes it's true I will fix it